.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.md linguist-detectable=true
+*.md linguist-documentation=false
```

Comment:
What do these changes do?

Reply:
do you know how if you for example write js code and push it to a github repo? And then the repo has a js tag. Well this makes it detect markdown(md) as a language.